.pre-commit-config.yaml

```
@@ -0,0 +1,17 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+      - id: check-merge-conflict
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  - repo: https://github.com/commitizen-tools/commitizen
+    rev: v2.16.0
+    hooks:
+      - id: commitizen
+        stages: [commit-msg]
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.2.1
+    hooks:
+      - id: prettier
```

@phillipjf Add a prettier exclusion for `CHANGELOG.md` because the auto-generated one that comes from `semantic-release` isn't pre-formatted - otherwise prettier will try to reformat it.

```suggestion
      - id: prettier
        exclude: CHANGELOG.md
```